packages/ui/components/Table/Table.tsx

```diff
@@ -12,4 +12,8 @@ export const Table = React.forwardRef<HTMLTableElement, React.HTMLAttributes<HTM
   )
 )
 
+export const TableFooter = ({ children }: { children: React.ReactNode }) => {
+  return <tfoot>{children}</tfoot>
```

Comment:
no need for return this can be one liner ie:

`
export const TableFooter = ({ children }: { children: React.ReactNode }) => <tfoot>{children}</tfoot>`